page-tracking/src/hw_mem_map.rs

```diff
@@ -146,6 +147,96 @@ impl HwMemRegion {
         let pages = self.size / PageSize::Size4k as u64;
         self.base.checked_add_pages(pages).unwrap()
     }
+
+    /// Returns the page size if this has been partitioned, or None if it hasn't
+    pub fn page_size(&self) -> Option<PageSize> {
+        self.page_size
+    }
+
+    /// break a `HwMemRegion` into multiple `HwMemRegion`s that can all fit into vm pages of the same size
```

Comment:
Nit: Capitalize first letter here and elsewhere in the file.

Reply:
Fixed